src/transferHandlers/progress.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as Comlink from 'comlink';
+
+Comlink.transferHandlers.set('penumbra-progress', {
+  /** Checks if object is a penumbra-progress event */
```

Comment:
Can you document function params and return -- `@param object - ..., @returns ...`

Reply:
Doing this now.